docs/index.rst

```diff
@@ -69,6 +69,53 @@ Example:
    to this mount.
 
 
+CDN
+~~~
+
+Crane also supports some advanced options for v2 content that resides on a Content
+Delivery Network.
+
+The URL match and replace parameters support rewriting the URL where content is
+served on the CDN (redirect URL) to a new location. This can be used independently
+of the authorization features if desired.
+
+The URL authorization options allow Crane to support URL-based token authorization
+via Akamai's `Authorization Token 2.0 <https://community.akamai.com/people/B-3-2THQ57/blog/2016/02/26/ever-wondered-how-to-protect-your-assets-with-token-auth-edgeauth-20>`_
+specification. When authorization is used, Crane will append a query string parameter
+to all v2 redirects containing an expiration time and HMAC token. The field delimiter
+in the token is set to ``~`` based on Akamai's specificaton.
+
+Example URL with authorization token:
+
+::
+
+  https://cdn.example.com/path/to/content?_auth_=exp=1524660654~hmac=d039ac10e019fd13824a3f861b4f55df40e2a402d102b5266194fff6f3a24ed0
+
+The following options should go under a section named ``[cdn]``
+
+url_match
+  optional portion of URL to match when rewriting content redirect URLs
+
+url_replace
+  optional replacement for ``url_match`` when rewriting the redirect URL. This will
+  only be used when specified along with ``url_match``
+
+url_auth_secret
+  seed used for generating HMAC tokens for URL authorization. This can be any valid
+  hexadecimal string (e.g. b8470f032e4ea2aa). This must match the secret configured
+  on the Akamai property where content is served.
+
+url_auth_param
+  name of query string parameter to hold authorization token. This must match the
+  parameter configured on the Akamai property. Defaults to ``_auth_``
+
+url_auth_ttl
+  window in seconds that authorized URLs generated by Crane will remain valid.
+  Defaults to ``300``
+
+url_auth_algo
```

Comment:
i thin we should add validation here as well.
```
In [10]: hashlib.algorithms_guaranteed
Out[10]: {'md5', 'sha1', 'sha224', 'sha256', 'sha384', 'sha512'}
```
As far as i understand we should limit those to sha256,sha1 and md5 since with other algorithms Akamai is not working right?